gems/pending/appliance_console/utilities.rb

```diff
@@ -49,8 +49,7 @@ def self.db_host_type_database
     end
 
     def self.pg_status
-      system("service #{PostgresAdmin.service_name} status > /dev/null 2>&1")
-      $?.exitstatus == 0 ? "running" : "not running"
+      AwesomeSpawn.run("pg_isready", :params => ["-q"]).success? ? "running" : "not running"
```

Comment:
No, I'm saying that I like the way you changed it because we don't really care if the service is running here, but rather if we can connect to the database or not.  What I meant was that we really needed to change away from `service` to `systemctl` and `LinuxAdmin::Service` handles that for you here https://github.com/ManageIQ/linux_admin/blob/master/lib/linux_admin/service.rb#L37